### DISCUSSION TITLE
关于对`ScriptCat`脚本管理器兼容性测试

### DISCUSSION THREAD

**Discussion by maboloshi** [maboloshi/github-chinese#147] [General]
[ScriptCat](https://docs.scriptcat.org/) 也是一个第三方脚本管理器
谁有兴趣可以尝试进行兼容性测试

**Comment by TC999** [comment]
# 脚本猫测试结果
## Chrome
![image](https://github.com/maboloshi/github-chinese/assets/88823709/755b058e-8cfa-44ad-afe3-15c81c3eae35)

## Firefox
![Firefox](https://github.com/maboloshi/github-chinese/assets/88823709/272fd2fd-a4bc-471e-89d2-0b469cf8f519)

**Comment by gaenyong** [comment]
大佬，可不可以兼容一下**Userscripts**，stay太臃肿，macaque还要钱，感谢🙏

**Reply by maboloshi** [reply]
没有mac 没法兼容性测试🤣

**Reply by gaenyong** [reply]
噢噢 那兼容性测试我能做么 有没有教程 我纯小白 只会用不会做hh

**Reply by gaenyong** [reply]
> 没有mac 没法兼容性测试🤣

我之前把脚本加到userscripts里了，它能识别GitHub，启用了，但界面没有变化，还是全英文。

**Comment by TC999** [comment]
刚测试一下，正常使用没问题，但是不能用于开发

原因：本地测试可以正常读取本地文件，但是本地文件更改后的内容脚本不会读取

**Comment by maboloshi** [comment]
不打算兼容
